examples/pip_parse/requirements.in

```diff
@@ -1,3 +1,5 @@
 requests~=2.25.1
 s3cmd~=2.1.0
 yamllint~=1.28.0
+apache-airflow
+apache-airflow-providers-sqlite
```

Comment:
Maybe we could use the following cycle as an example instead?
```
.-> @docs_deps_sphinx//:pkg
|   @docs_deps_sphinxcontrib_serializinghtml//:pkg
`-- @docs_deps_sphinx//:pkg
```

It could be much more lightweight. Some of `airflow` deps do not have wheels and having a smaller example may be better.

Reply:
I'll give this a shot. I have a sneaking suspicion that the windows build problems are due to wheels / builds somehow so going for a purelib cycle would make for a better example.